src/modules/edit_task.js

```diff
@@ -0,0 +1,13 @@
+const editingTask = () => {
+  const element = document.querySelectorAll('.text');
+  element.forEach((ele) => {
+    ele.addEventListener('click', () => {
+      ele.contentEditable = true;
+      //   ele.style.backgroundColor = 'lightyellow';
+      ele.parentElement.parentElement.lastElementChild.previousElementSibling.classList.add('hidden');
+      ele.parentElement.parentElement.lastElementChild.classList.remove('hidden');
+    });
+  });
+};
```

Comment:
- [ ] Well done with the edit. As mentioned in the requirements,  all changes in your todos should be saved in local storage. kindly save the todo in local storage after exit